client/landing/gutenboarding/onboarding-block/features/data.tsx

```diff
@@ -0,0 +1,161 @@
+				/>
+			</SVG>
+		),
+	},
+	[ FEATURE_SEO ]: {
+		id: FEATURE_SEO,
+		name: translate( 'SEO tools' ) as string,
+		description: translate( 'Boost your SEO and connect a Google Analytics account.' ) as string,
+		minSupportedPlan: PLAN_BUSINESS,
+		icon: (
+			<SVG viewBox="0 0 24 24">
+				<Path d="M8.1437 15.8563H5.57227V9.42773H8.1437V15.8563Z" />
+				<Path d="M13.2862 15.8571H10.7148V3H13.2862V15.8571Z" />
+				<Path d="M18.4289 15.8574H15.8574V6.85742H18.4289V15.8574Z" />
+				<Path fillRule="evenodd" clipRule="evenodd" d="M21 20.7148H3V18.7148H21V20.7148Z" />
+			</SVG>
+		),
+	},
+	[ FEATURE_PLUGINS ]: {
+		id: FEATURE_PLUGINS,
+		name: translate( 'Plugins' ) as string,
+		description: translate( 'Install plugins to extend the power of your site.' ) as string,
+		minSupportedPlan: PLAN_BUSINESS,
+		icon: (
+			<SVG viewBox="0 0 24 28">
+				<Path
+					fillRule="evenodd"
+					clipRule="evenodd"
+					d="M4 2.27539C2.78 3.32215 2 5.11415 2 6.8265C2 9.34099 3.79 11.3776 6 11.3776C6.387 11.3776 6.753 11.2866 7.093 11.1364C7.686 11.9761 8.28 12.8158 8.9 13.6304L10.978 11.2638C10.264 10.5584 9.528 9.8803 8.788 9.20446C8.92 8.81989 9 8.40346 9 7.96428C9 6.07898 7.657 4.55095 6 4.55095C4.69 4.55095 4 3.23681 4 2.27539ZM15.492 13.0934L14.152 14.618C16.08 16.7627 17.905 19.0258 19.6 21.4185L20.75 23.0398L19.25 24.7465L17.825 23.4403C15.722 21.5095 13.733 19.4331 11.848 17.2394L5 25.031L3 22.7554L13.493 10.8167C13.188 10.1261 13 9.35806 13 8.53317C13 5.70579 15.015 3.41317 17.5 3.41317C18.225 3.41317 18.9 3.62707 19.507 3.97409L16 7.96428L18 10.2398L21.507 6.24965C21.812 6.94028 22 7.70828 22 8.53317C22 11.3605 19.985 13.6532 17.5 13.6532C16.775 13.6532 16.099 13.4393 15.492 13.0934Z"
+				/>
+				<Path
+					fillRule="evenodd"
+					clipRule="evenodd"
+					d="M4 2.27539C2.78 3.32215 2 5.11415 2 6.8265C2 9.34099 3.79 11.3776 6 11.3776C6.387 11.3776 6.753 11.2866 7.093 11.1364C7.686 11.9761 8.28 12.8158 8.9 13.6304L10.978 11.2638C10.264 10.5584 9.528 9.8803 8.788 9.20446C8.92 8.81989 9 8.40346 9 7.96428C9 6.07898 7.657 4.55095 6 4.55095C4.69 4.55095 4 3.23681 4 2.27539ZM15.492 13.0934L14.152 14.618C16.08 16.7627 17.905 19.0258 19.6 21.4185L20.75 23.0398L19.25 24.7465L17.825 23.4403C15.722 21.5095 13.733 19.4331 11.848 17.2394L5 25.031L3 22.7554L13.493 10.8167C13.188 10.1261 13 9.35806 13 8.53317C13 5.70579 15.015 3.41317 17.5 3.41317C18.225 3.41317 18.9 3.62707 19.507 3.97409L16 7.96428L18 10.2398L21.507 6.24965C21.812 6.94028 22 7.70828 22 8.53317C22 11.3605 19.985 13.6532 17.5 13.6532C16.775 13.6532 16.099 13.4393 15.492 13.0934Z"
+				/>
+			</SVG>
+		),
+	},
+	[ FEATURE_AD_FREE ]: {
+		id: FEATURE_AD_FREE,
+		name: translate( 'Ad-free' ) as string,
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Ad-free sites' )` *ES Score: 6*